src/components/ExtensibilitySpectrum/index.jsx

```diff
@@ -0,0 +1,177 @@
+                            styles.svgStrokeBlue
+                        )}
+                        d="M125.6 176l-29.2-20.8v-5.1l29.2-22.6v9.2l-21.3 15.7v.2l21.3 14.2v9.2zM154.9 126.1l-16.6 59.2h-5l16.5-59.2h5.1zM192.3 154.9l-29.2 20.8v-9.2l21.4-14.1v-.3l-21.4-15.7v-9.2l29.2 22.5v5.2z"
+                    />
+                </g>
+            </g>
+        </g>
+    </switch>
+);
+
+function SvgLineWithArrow() {
+    return (
+        <svg
+            className={styles.svgDashedLine}
+            height="80"
+            xmlns="http://www.w3.org/2000/svg"
+        >
+            <defs>
+                <marker
+                    className={classNames(
+                        styles.svgFillBlue,
+                        styles.svgStrokeBlue
+                    )}
+                    id="arrow"
+                    viewBox="0 0 10 10"
+                    refX="5"
+                    refY="5"
+                    markerWidth="2.5"
+                    markerHeight="2.5"
+                    orient="auto-start-reverse"
+                >
+                    <path d="M 0 0 L 10 5 L 0 10 z" />
+                </marker>
+            </defs>
+            <line
+                className={classNames(styles.arrowLine, styles.svgStrokeBlue)}
+                strokeDashoffset="-2.5"
+                strokeDasharray="10,5"
+                x1="10"
+                y1="10"
+                x2="10"
+                y2="60"
+                markerEnd="url(#arrow)"
+            />
+        </svg>
+    );
+}
+
+function IconContainer(props) {
+    const circleRadius = 2.5;
+
+    return (
+        <a className={styles.iconContainer} href={props.linkUrl}>
+            <svg xmlns="http://www.w3.org/2000/svg">
+                <circle
+                    className={styles.iconCircle}
+                    cx={`${circleRadius}em`}
+                    cy={`${circleRadius}em`}
+                    r={`${circleRadius}em`}
+                />
+                <svg
+                    height={`${circleRadius * 2}em`}
+                    width={`${circleRadius * 2}em`}
+                >
+                    {props.children}
+                </svg>
+            </svg>
+            <h2 className={styles.iconLabel}>{props.iconLabel}</h2>
+        </a>
+    );
+}
+
+export default function ExtensibilitySpectrum({
+    configLink,
+    workflowLink,
+    customWorkflowLink,
+    customCodeLink,
+}) {
+    return (
+        <div className={styles.container}>
+            <IconContainer
+                iconLabel="Application Configuration"
+                linkUrl={configLink}
+            >
+                <svg viewBox="-10 -10 300 300">{appConfigSvg}</svg>
+            </IconContainer>
+            <SvgLineWithArrow />
+            <IconContainer
+                iconLabel="Geocortex Workflow"
+                linkUrl={workflowLink}
+            >
+                <svg viewBox="-12 -11 48 48">{workflowSvg}</svg>
```

Comment:
Let's move the `svg` element into the icons themselves. The `viewBox` is highly important for it to render correctly.

Reply:
I took the SVG out in order to reuse the workflow and custom code SVG 

Those two chunks of SVG are used with different viewboxes in the two workflow icons